charts/pxc-db/README.md

```diff
@@ -67,12 +67,22 @@ The chart can be customized using the following configurable parameters:
 | `pxc.persistence.size` | Sets K8S persistent storage size for all PXC Pods | `8Gi`                      |
 | `pxc.disableTLS` | Disable PXC Pod communication with TLS | `false`                      |
 | `pxc.certManager` | Enable this option if you want the operator to request certificates from `cert-manager` | `false`                      |
+| `pxc.readinessProbes.failureThreshold` | When a probe fails, Kubernetes will try failureThreshold times before giving up | `5`                      |
```

Comment:
New options added in `1.9.0` should be added (`pxc.expose` and `pxc.replicationChannels`): https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L39-L55

Reply:
done